_dashboards/dev-tools/index-dev.md

```diff
@@ -7,14 +7,16 @@ has_children: true
+Interact directly with OpenSearch by using **Dev Tools** to set up your OpenSearch Dashboards environment, run queries, explore data, and debug problems. You can use the Dev Tools console to: 
 
-- Set up your OpenSearch Dashboards environment. For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 
-- [Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/). For example, you can use the console to run a query to find all the documents in your index that contain a specific word.
-- Debug problems with your queries. For example, if your query is not returning the results you expect, you can use the console to look for error messages and identify the problem.
-- Learn about the APIs in OpenSearch. For example, you can use the API reference documentation linked in the console (select the question circle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/question-circle.png" class="inline-icon" alt="question circle icon"/>{:/})) to look up the syntax for different API calls.
-- Develop custom visualizations. For example, you can use the console to create Vega visualizations. 
-- Customize the appearance and behavior of dashboards. For example, you can use the console to customize dashboard visualization colors or to add new filters. 
-- Identify and fix bugs. For example, you can use the console to view logs and identify the cause of the problem.
+- **Set up your OpenSearch Dashboards environment.** For example, you can use the console to configure authentication settings for your OpenSearch Dashboards instance. 
+- **[Run queries to explore your data]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/run-queries/).** For example, you can use the console to run a query to find all the documents in your index that contain a specific word.
+- **Debug problems with your queries.** For example, if your query is not returning the results you expect, you can use the console to look for error messages and identify the problem.
+- **Learn about the APIs in OpenSearch.** For example, you can use the API reference documentation linked in the console (select the question circle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/question-circle.png" class="inline-icon" alt="question circle icon"/>{:/})) to look up the syntax for different API calls.
+- **Develop custom visualizations.** For example, you can use the console to create Vega visualizations. 
+- **Customize the appearance and behavior of dashboards.** For example, you can use the console to customize dashboard visualization colors or to add new filters. 
+- **Identify and fix bugs.** For example, you can use the console to view logs and identify the cause of the problem.
```

Comment:
I'm not sure I understand what this bullet point means. What kinds of logs? Dashboards logs or user custom logs? And bugs where?

Reply:
Deleted. If dev provides more details, I'll add later.